dynamiq/nodes/tools/http_api_call.py

```diff
@@ -85,9 +85,9 @@ def execute(self, input_data: HttpApiCallInputSchema, config: RunnableConfig = N
         self.run_on_node_execute_run(config.callbacks, **kwargs)
         data = input_data.data
 
-        url = self.connection.url
-        if url_path := input_data.url_path:
-            url = urljoin(url, url_path)
+        url = input_data.url or self.url or self.connection.url
+        if url is None:
```

Comment:
Let's maybe cover empty string too 
```suggestion
        if not url:
```